crates/storage/src/managers/mod.rs

```diff
@@ -1,2 +1,37 @@
+use std::sync::Arc;
+
+use strata_db::traits::Database;
+
 pub mod checkpoint;
 pub mod l2;
+
+/// A consolidation of database managers and ops. Ideally ops should not be exposed.
+#[derive(Clone)]
+pub struct DbManager {
+    l2_block_manager: Arc<l2::L2BlockManager>,
+    checkpoint_manager: Arc<checkpoint::CheckpointDbManager>,
+}
+impl DbManager {
```

Comment:
Not really a manager since it's just a dumb container for other data.  It makes sense to put this all under an umbrella type something like "`NodeStorageInterface`" and not in the `managers` module.

Should also have a separate one of these for the sequencer databases.

Reply:
We need a good name for the container of similar items. :)

Reply:
But as I think more about it, I think it is fine. It is something similar to the `Database` trait isn't it?

Reply:
Structurally yes, but it's not serving a comparable role to make sense to be called that the `Database`-related traits aren't managers either, they're more like "providers".  The term "manager" implies there's some nontrivial amount of underlying complexity being *managed*, but it's not, it's *just* a container for the things that actually do that.

Reply:
I see. Probably `ManagersContainer` then?

> and not in the managers module.

Why not? I see the container to be just a container that will be passed around other services and tasks. It won't have different and extra functionalities of its own compared to what the current managers have.

Reply:
How about `NodeStorage`?

Reply:
That's better 👍